### DISCUSSION TITLE
Could libxev be of interest for uSockets?

### DISCUSSION THREAD

**Discussion by ddevienne** [uNetworking/uWebSockets#1691] [Ideas]
Hi. There's no Discussions tab on `uSockets`, so I'm asking here (its sibling project) instead.

I see Bun uses https://github.com/mitchellh/libxev since recently.
Could that be a new backend of uSockets in the future?
Or possibly completely replace uSockets even?

Just wondering. Thanks, --DD

PS: Still no Windows support in libxev yet, but it's planned.
PPS: Just noticed it's from Mitchell Hashimoto, which IMHO leads some credence as to the quality

**Comment by uNetworkingAB** [comment]
If you want to add a new event loop backend to uSockets you can look at how the others are done, there are a few already, but it's up to you because I have no use for such a thing right now
